Hyper/Window.cs

```diff
@@ -116,9 +73,9 @@ protected override void OnUpdateFrame(FrameEventArgs e)
                 Close();
             }
 
-            UpdateCamera(input, time);
+            _scene.UpdateCamera(input, time, new Vector2(MouseState.X, MouseState.Y));
```

Comment:
I feel like we're exposing too much details about the scene. Maybe something like `_scene.Update(<all the necessary stuff>)` but this is ugly too. If I think of a better solution I'll push it in my commit

Reply:
Maybe add structs like MouseArguments, TimeArguments and so on and them pass them to the Update() method?